core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
Sry, Duarte, double posting: as you can [read here](https://groups.google.com/d/msg/tiddlywiki/7LfTfO7oC9U/OkH7Zz--EQAJ) I had quite a weird misunderstanding about the ways the **[img[]]** markup works.